src/bidiMapper/domains/network/NetworkManager.ts

```diff
@@ -43,12 +43,19 @@ export class NetworkManager {
     return this.#cdpTarget;
   }
 
-  #getOrCreateNetworkRequest(id: Network.Request): NetworkRequest {
+  /**
+   * Gets the network request with the given ID, if any.
+   * Otherwise, creates a new network request with the given ID and cdp target.
+   */
+  #getOrCreateNetworkRequest(
+    id: Network.Request,
+    cdpTarget: CdpTarget
```

Comment:
should this always be `this.#cdpTarget`?

Reply:
Good point. I think that should work as well and would simplify a few things. (Although, note the comment above still remains open in case you have a suggestion to inject CdpTarget into NetworkProcessor directly.)

Reply:
Done